components/display/tm1637.rst

```diff
@@ -43,6 +43,37 @@ Configuration variables:
 
 .. _display-tm1637_lambda:
 
+Keyboard scan (8x2bit) Binary-Sensor
+------------------------------------
+
+The TM1637 LED display controller also includes a special circuit with keyboard scan interface and enhanced identification circuit with anti-interference keys.
+This circuit uses the 8 segment lines that also drive the LED display. But combined with the K1 and K2 pins we can add 16 keys that can be used as binary sensors in esphome.
+
+.. figure:: images/tm1637_key_connections.png
+    :align: center
+    :width: 50.0%
+
+    TM1637 display keyboard connections
+
+.. code-block:: yaml
+
+    binary_sensor:
+      - platform: tm1637
+        id: key0
+        name: key1-00
+        tm1637_id: tm1637_display
+        key: 0
+
+Configuration variables:
+************************
+
+- **id** (*Optional*, :ref:`config-id`): Set the ID of this sensor.
+- **name** (*Optional*, string): The name for the binary sensor.
+- **tm1637_id** (**Required**, ::ref:`config-id`): The id of the tm1637 that should be used to scan the keys.
```

Comment:
```suggestion
- **tm1637_id** (*Optional*, ::ref:`config-id`): The id of the tm1637 that should be used to scan the keys in case you are using multiple devices.
```